extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -423,10 +430,10 @@ export class AmpA4A extends AMP.BaseElement {
   preconnectCallback(unusedOnLayout) {
     this.preconnect.preload(this.getSafeframePath_());
     this.preconnect.preload(getDefaultBootstrapBaseUrl(this.win, 'nameframe'));
-    if (!this.config) {
+    const preconnect = this.getPreconnectUrls();
+    if (preconnect.length == 0) {
```

Comment:
I would argue this is unnecessary.  If its an empty array it would result in no preconnects